packages/calypso-products/src/is-difm-extra-page-product.ts

```diff
@@ -0,0 +1,7 @@
+import { camelOrSnakeSlug } from './camel-or-snake-slug';
+import { WPCOM_DIFM_EXTRA_PAGE } from './constants';
+import type { WithCamelCaseSlug, WithSnakeCaseSlug } from './types';
+
+export function isDIFMExtraPageProduct( product: WithCamelCaseSlug | WithSnakeCaseSlug ): boolean {
+	return camelOrSnakeSlug( product ) === WPCOM_DIFM_EXTRA_PAGE;
+}
```

Comment:
The `src` folder has a surprisingly massive amount of random utility functions. This is ordered by some convention it seems (a lot of `get-...` and `is-...` stuff in there). I guess that's ok, or do you think there are ways to organize things a little. Just poking for thought if you think similarly, no need to move anything around. 🤔

Reply:
FWIW these functions were migrated from inside calypso to this package to help organize and clean them up. I removed as many as I could at that time but as you can see there are still a ton. In many cases simply importing the product slug itself and doing a comparison at the call-site is sufficient, but these functions persist because we have different definitions of "product" (cart product, products list product, subscription product, receipt product, etc.) that get passed to them. Eventually it would be nice to remove all these.